core/src/main/java/uk/gov/gchq/koryphe/util/JavaUtils.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2022 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.koryphe.util;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+
+/**
+ * Copies of various methods/class from the Java 11 JDK.
+ */
+public final class JavaUtils {
```

Comment:
Could be worth making it clearer that this is a temporary proxy of the Java 11 methods, and perhaps flagging with a warning/deprecated flag.